sormas-api/src/main/java/de/symeda/sormas/api/event/EventParticipantDto.java

```diff
@@ -1,20 +1,17 @@
-/*******************************************************************************
+/*
```

Comment:
Please also update the copyright year to 2016-2021

Reply:
done